redaxo/src/addons/media_manager/install.php

```diff
@@ -11,6 +11,8 @@
  * @var rex_addon $this
  */
 
-if (!$this->hasConfig('jpg_quality')) {
+if (!$this->hasConfig()) {
     $this->setConfig('jpg_quality', 85);
+    $this->setConfig('png_compression', 6);
+    $this->setConfig('interlace', false);
```

Comment:
Wofür seid ihr?
1. Interlace hier default auf true setzen. Also als Standard zukünftig global aktivieren
2. Globale Einstellung default aus lassen, aber bei den Core-Types per Effekt aktivieren
3. Globale Einstellung default aus lassen, auch in Core-Types aus lassen

Bei 1. und 2. könnte man es auch so machen, dass es bei Updates deaktiviert bleibt (gab es vorher ja nicht), bei Neu-Installationen aber aktiv ist.